test/app.locals.js

```diff
@@ -1,26 +1,25 @@
+'use strict'
 
+var assert = require('assert')
 var express = require('../')
 
 describe('app', function(){
-  describe('.locals(obj)', function(){
-    it('should merge locals', function(){
-      var app = express();
-      Object.keys(app.locals).should.eql(['settings']);
-      app.locals.user = 'tobi';
-      app.locals.age = 2;
-      Object.keys(app.locals).should.eql(['settings', 'user', 'age']);
-      app.locals.user.should.equal('tobi');
-      app.locals.age.should.equal(2);
+  describe('.locals', function () {
+    it('should default object', function () {
+      var app = express()
```

Comment:
**Issue**: The removal of tests for '.locals(obj)' functionality could lead to regressions if this feature is still supported. The new tests do not cover the merging behavior of locals. <br> **Fix**: Restore or modify the existing tests to ensure that the merging behavior of locals is still validated if this feature is still intended to be supported. <br> **Code Suggestion**: 
 ```
 -  describe('.locals', function () {
 +  describe('.locals(obj)', function () {
      it('should default object', function () {
        var app = express()
        assert.ok(app.locals)
        assert.strictEqual(typeof app.locals, 'object')
 -      })
 +      }),
 +      it('should merge locals', function(){
 +        var app = express();
 +        Object.keys(app.locals).should.eql(['settings']);
 +        app.locals.user = 'tobi';
 +        app.locals.age = 2;
 +        Object.keys(app.locals).should.eql(['settings', 'user', 'age']);
 +        app.locals.user.should.equal('tobi');
 +        app.locals.age.should.equal(2);
 +      })
 ```